geospaas/base_viewer/templates/base_viewer/page_structure.html

```diff
@@ -0,0 +1,33 @@
+{% load static %}
+{% load leaflet_tags %}
+<html>
+
+<head>
+	<title>Meta-data finder</title>
+	{% leaflet_js plugins="ALL" %}
+	{% leaflet_css plugins="ALL" %}
+<link rel="stylesheet" href="{% static 'base_viewer/styles.css' %}" >
+
+</head>
+
+<body class="body" >
+	<div id="para_container">
+
+		<div id=content class="para2"><!--start of contentmap-->
+			{% block contentmap %}
+			{% endblock contentmap%}
+		</div><!--end of contentmap-->
+
+		<div id=result class="para2"><!--start of result section-->
+			{% block content_result %}
+			{% endblock content_result%}
+		</div><!--end of result section-->
+
+		<div id="searchbox"><!--start of searchbox-->
+			{% block searchbox %}
+			{% endblock searchbox %}
+		</div><!--end of searchbox-->
```

Comment:
Where is `searchbox` defined? I could find only `content_result` and `contentmap` in `elements.html`

Reply:
It is not used at all. Maybe used for ADAS viewer. Right now, it is just an empty tag of HTML

Reply:
Can you remove it? If we need it in ADAS, it is better to extend page_structure.html and add something than have an empty, potentially-non needed placeholder.